app/Http/Controllers/Admin/HomeController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Models\User;
+use App\Models\Post;
+use App\Models\Borrow;
+use App\Models\Book;
+use App\Models\BorrowDetail;
+use Carbon\Carbon;
+use DB;
+
+class HomeController extends Controller
+{
+    /**
+     * Display data in statistical page.
+     *
+     * @return data
+     */
+    public function index()
+    {
+        $data['users'] = User::all();
```

Comment:
mấy cái này em count ở đây, qua bên kia chỉ việc in ra thôi. 
nếu làm thế này vừa tốn query, performance ko tốt nữa. 
đổi hết giúp a mấy chỗ count nhé

Reply:
đặt tên `users ` -> `users_count`